Library/Homebrew/utils/github.rb

```diff
@@ -476,6 +476,15 @@ def sponsors_by_tier(user)
     open_api(url, scopes: ["admin:org", "user"], data: data, request_method: "POST")
   end
 
+  def get_repo_license(user, repo)
+    res = GitHub.open_api("#{GitHub::API_URL}/repos/#{user}/#{repo}/license")
+    return unless res.key?("license")
+
+    res["license"]["spdx_id"] || nil
```

Comment:
```suggestion
    res["license"]["spdx_id"]
```

as `nil` is default.